recipe/meta.yaml

```diff
@@ -22,13 +22,14 @@ requirements:
     - ipykernel
     - traitlets
     - notebook
+    - widgetsnbextension
 
 test:
   imports:
     - ipywidgets
 
 about:
-  home: http://github.com/jakevdp/ipywidgets
+  home: https://github.com/ipython/ipywidgets
   license: BSD 3-clause
   summary: IPython Static Widgets
```

Comment:
The front-end is now kernel-language agnostic and we are working hard to make it work outside of the notebook....